### ISSUE TITLE
RBM : Learning rate

### ISSUE THREAD

**Issue by FrancoisLasson** [FrancoisLasson/Temporal_DBN#5]
This issue answers #1

In order to know the influence of the learning rate, we train several RBMs in the same conditions using the learning rate as factor. We check results (cost evolution) for 3 values of the LR: 1e-2, 1e-3 and 1e-4.

_Condition
number of epochs = 10000, number of hidden neurones = 30, batch_size=100
9 gestures, without shuffle_

Results are reported in the following plots :
• 1e-2
![rbm_1459935120 99](https://cloud.githubusercontent.com/assets/11289891/14345339/8be16c86-fcac-11e5-8056-733830648bcc.png)
• 1e-3
![rbm_1459877678 13](https://cloud.githubusercontent.com/assets/11289891/14345338/8be11196-fcac-11e5-8b58-2c848570e8f7.png)
• 1e-4
![rbm_1459941680 58](https://cloud.githubusercontent.com/assets/11289891/14345340/8be18950-fcac-11e5-89bd-8b0d0359a3f6.png)

As we could see in these graph, the learning rate have an influence on the cost decay rate. In fact, the learning rate constant is a step that influence the learning speed. It's also could see as a step used to travel along the Error function. So, a small learning rate is better suited to found the global minima but require more epoch.

At the moment, we consider that 1e-3 is the best learning rate in our case. Yet, we will have to compare 1e-3 and 1e-4 on the PER result.
